Jenkinsfile

```diff
@@ -173,6 +173,64 @@ pipeline {
                 '''
             }
         }
+        stage('Publish binaries') {
+            when {
+                branch 'master'
+            }
+            agent {
+                dockerfile {
+                    filename 'Dockerfile'
+                    additionalBuildArgs  '--target ci-go'
+                    registryCredentialsId 'dockerhub-pro-credentials'
+                }
+            }
+            steps {
+                withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'nexus', usernameVariable: 'NEXUS_USR', passwordVariable: 'NEXUS_PSW']]) {
+                    script {
+                        def platforms = [ "windows/amd64", "linux/amd64", "darwin/amd64", "darwin/arm64" ]
```

Comment:
I came late, but we need to add the "universal" platform too or modify the consumer to use a specific platform to handle this use case. cc. @glodieu @lbauret

Reply:
No need for the amf-custom-validator to have a universal build. Universal build is an ACB extension concept. Under the hood, the universal build will use the linux amf-custom-validator build.